### PR TITLE
Facility overview now shows verification status of datafiles.

### PR DESCRIPTION
**Pull request by wettenhj**
verified property of DataFile model now returns False if a
datafile doesn't have any DataFileObjects.

### CODE REVIEW

File: tardis/tardis_portal/views.py [hunk 1]
```diff
@@ -425,13 +425,33 @@ def datetime_to_us(dt):
...
+            else:
+                verified = "No"
+                try:
+                    file_object_size = \
+                        datafile.file_object.size \
+                        if datafile.file_object else 0
+                    if file_object_size < int(datafile.size):
+                        verified = "No (%s of %s bytes uploaded)" \
+                            % ('{:,}'.format(file_object_size),
+                               '{:,}'.format(int(datafile.size)))
```
Comment (grischa) [inline tardis/tardis_portal/views.py]: Just wondering about the purpose of `'{:,}'.format()`, as I haven't used the format method much.

Comment (wettenhj) [inline tardis/tardis_portal/views.py]: It just uses commas to group digits into 3, e.g. 123,456 of 125,000 bytes uploaded: https://docs.python.org/2/library/string.html#formatspec

Comment (grischa) [inline tardis/tardis_portal/views.py]: Cool, thanks!

Comment (grischa) [inline tardis/tardis_portal/views.py]: No need to change it, views.py is a mess anyway, but I'm still wondering why you didn't just write `"No ({:,} of {:,} bytes uploaded)".format(file_object_size, int(datafile.size))`?

**Comment by grischa** [comment]
Thanks, @wettenhj

File: tardis/tardis_portal/views.py (hunk 1 above)
Comment (wettenhj) [inline tardis/tardis_portal/views.py]: Yes, that looks more elegant.  No there wasn't a good reason why I didn't write it that way.  Feel free to change it.

**Pull request merged**